src/algebraic_topology/dold_kan/functor_n.lean

```diff
@@ -0,0 +1,66 @@
+/-
+Copyright (c) 2022 Joël Riou. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Joël Riou
+-/
+
+import algebraic_topology.dold_kan.p_infty
+
+/-!
+
+# Construction of functors N for the Dold-Kan correspondence
+
+TODO (@joelriou) continue adding the various files referenced below
+
+In this file, we construct the functors `N₁ : simplicial_object C ⥤ karoubi (chain_complex C ℕ)`
+and `N₂ : karoubi (simplicial_object C) ⥤ karoubi (chain_complex C ℕ)`
```

Comment:
Do these functors have names in the literature? Or are they always called `N₁` and `N₂`?

Reply:
These functors are not in the literature! I have added a note regarding the indices (which reflects the number of occurences of `karoubi` at the source/target.